bootstrap.sh

```diff
@@ -246,15 +252,29 @@ if [ -d appscale/.appscale/certs ]; then
 
 
         # Let's upgrade the repository: if GIT_TAG is empty we are on HEAD.
-        if [ "$FORCE_UPGRADE" = "Y" ]; then
+        if [ "$STASH" = "Y" ]; then
                 echo "It is a force upgrade, so a git stash will be performed."
                 (cd appscale; git stash save -u)
                 (cd appscale-tools; git stash save -u)
         fi
 
         if [ -n "${GIT_TAG}" ]; then
+                set +e
                 (cd appscale; git checkout "$GIT_TAG")
+                if [ $? -gt 0 ]; then
+                    echo "Please stash your local unsaved changes."
```

Comment:
Change these three prints to:

echo "Please stash your local unsaved changes and checkout the version of AppScale you are currently using to fix this error."
echo "e.g.: `git stash save -u; git checkout <AppScale_version>`"


The last sentence is too specific for the bootstrap.